pkg/kubectl/cmd/resource/get.go

```diff
@@ -209,10 +209,19 @@ func (options *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args
 }
 
 // Validate checks the set of flags provided by the user.
-func (options *GetOptions) Validate() error {
+func (options *GetOptions) Validate(cmd *cobra.Command) error {
 	if len(options.Raw) > 0 && (options.Watch || options.WatchOnly || len(options.LabelSelector) > 0 || options.Export) {
 		return fmt.Errorf("--raw may not be specified with other flags that filter the server request or alter the output")
 	}
+	if cmdutil.GetFlagBool(cmd, "show-labels") {
+		invalidOutputOptions := [5]string{"yaml", "json", "name", "jsonpath", "jsonpath-file"}
```

Comment:
how about we do whitelisting instead of blacklisting?
for example you forgot to add go-template here (?) and perhaps if we add a format like toml in the future, that won't be included here either...

--show-labels works with pretty much only table/wide printers. so I think whitelisting is better.

Reply:
Modified. @ahmetb PTAL.